src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -14787,6 +14787,7 @@ public static partial class Utf8
         public static System.Buffers.OperationStatus ToUtf16(System.ReadOnlySpan<byte> source, System.Span<char> destination, out int bytesRead, out int charsWritten, bool replaceInvalidSequences = true, bool isFinalBlock = true) { throw null; }
         public static bool TryWrite(System.Span<byte> destination, [System.Runtime.CompilerServices.InterpolatedStringHandlerArgumentAttribute("destination")] ref System.Text.Unicode.Utf8.TryWriteInterpolatedStringHandler handler, out int bytesWritten) { throw null; }
         public static bool TryWrite(System.Span<byte> destination, IFormatProvider? provider, [System.Runtime.CompilerServices.InterpolatedStringHandlerArgumentAttribute("destination", "provider")] ref System.Text.Unicode.Utf8.TryWriteInterpolatedStringHandler handler, out int bytesWritten) { throw null; }
+        public static bool IsValid(System.ReadOnlySpan<byte> value) { throw null; }
```

Comment:
Can you also search around dotnet/runtime for places this can be used?  For example, System.Text.Json currently does this with a `public static unsafe bool IsValidUtf8String(ReadOnlySpan<byte> bytes)` method that can be entirely replaced when targeting .NET 8+.

Reply:
One useful pattern to look for is for call sites that invoke `UTF8Encoding.GetCharCount` and throw away the return value. Those call sites are probably calling `GetCharCount` just for its side effect of throwing an exception when it sees invalid data.

Another might be to look for places which catch `EncoderFallbackException` and to see if they're actually trying to convert bytes \<-\> chars or if they're just performing validation.